packages/unmock-core/src/__tests__/serviceStore.test.ts

```diff
@@ -1,65 +1,135 @@
 import { serviceStoreFactory } from "../service";
 
-describe("State behaviour test suite", () => {
+describe("Fluent API and Service instantiation tests", () => {
   // define some service populators that match IOASMappingGenerator type
   const NoPathsServicePopulator = () => ({ petstore: {} });
   const EmptyPathsServicePopulator = () => ({ petstore: { paths: {} } });
   const PetsPathsServicePopulator = () => ({
     petstore: { paths: { "/pets": { get: {} } } },
   });
 
-  test("creating a state class", () => {
-    const state = serviceStoreFactory(NoPathsServicePopulator);
-    expect(state.noservice).toThrow("Can't find specification");
-    expect(state.petstore).toThrow("has no defined paths");
+  test("Store without paths", () => {
+    const store = serviceStoreFactory(NoPathsServicePopulator);
+    expect(store.noservice).toThrow("Can't find specification");
+    expect(store.petstore).toThrow("has no defined paths");
   });
 
-  test("state class with basic call to services", () => {
-    const state = serviceStoreFactory(EmptyPathsServicePopulator);
-    state.petstore(); // Should pass
+  test("Store with empty paths", () => {
+    const store = serviceStoreFactory(EmptyPathsServicePopulator);
+    expect(store.petstore).toThrow("has no defined paths");
+    expect(store.petstore.get).toThrow("has no defined paths");
```

Comment:
These now throw because it doesn't make sense to set a state for a service that has no endpoints. Same for the changes below.